Gemfile.lock

```diff
@@ -1,377 +0,0 @@
-GIT
```

Comment:
This file is required to ensure everyone has the same versions of gems , heroku will require it too